dev-util/packer/packer-1.4.3.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit golang-vcs-snapshot
+
+DESCRIPTION="A tool to create identical machine images for multiple platforms"
+HOMEPAGE="https://www.packer.io"
+
+EGO_PN="github.com/hashicorp/packer"
+SRC_URI="https://github.com/hashicorp/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MPL-2.0"
```

Comment:
Do I understand correctly that all vendored deps are linked into the final executable?

I see their licenses include BSD, ISC, MIT, Apache-2.0, MPL-2.0, GPL-3 (Go.Sed).

The first three shouldn't be a problem. I don't know about compatibility of Apache-2.0 and MPL-2.0 (@gentoo/licenses, help?). But GPL-3 is definitely the most restrictive of all. So if Apache-2.0 and MPL-2.0 are compatible, the package still must be licensed as GPL-3 because of Go.Sed.

I'll report this upstream once I get confirmation on Apache-2.0/MPL-2.0.

Reply:
Ok, FWIR MPL-2.0 is meant to be compatible with Apache-2.0. There's a whole guide on combining it with GPL: https://www.mozilla.org/en-US/MPL/2.0/combining-mpl-and-gpl/

Reply:
Filed https://github.com/hashicorp/packer/issues/8103

Reply:
I see. Weird.

Reply:
I'm surprised this is packaged in Debian. However, FWICS Go.Sed was added pretty recently, so it's possible that Debian has only versions prior to that.

Reply:
@mgorny 
So I performed some analysis on the repo using this tool: https://github.com/src-d/go-license-detector

and came up with the following list:

- 0BSD
- Apache-2.0
- BSD-2-Clause
- BSD-2-Clause-NetBSD
- BSD-3-Clause
- BSD-3-Clause-Clear
- BSD-3-Clause-No-Nuclear-License-2014
- BSD-4-Clause
- BSD-Source-Code
- ECL-2.0
- ICU
- ImageMagick
- ISC
- JSON
- MIT
- MIT-advertising
- MIT-feh
- MPL-2.0
- MPL-2.0-no-copyleft-exception
- UPL-1.0
- Xnet

the command used was:
` find -type f -name LICENSE\* | sed -e 's/LICENSE.*//' | xargs license-detector | egrep -v './' | grep -v 'no license' | awk '{print $2}' | sort | uniq`

Reply:
and here is the list that passes `repoman`

- Apache-2.0
- BSD-2
- BSD-4
- ECL-2.0
- ISC
- JSON
- MIT
- MPL-2.0

Reply:
(this is for packer-1.4.4, nominally without GPL-3)